epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityItem.java

```diff
@@ -55,31 +61,24 @@
    *
    * @param view        the current {@link com.airbnb.epoxy.EpoxyViewHolder}'s itemView
    * @param parent      the {@link androidx.recyclerview.widget.RecyclerView}
-   * @param vertical    true if it scroll vertically
    * @return true if the view has been measured
    */
-  boolean update(@NonNull View view, @NonNull RecyclerView parent,
-      boolean vertical, boolean detachEvent) {
+  boolean update(@NonNull View view, @NonNull RecyclerView parent, boolean detachEvent) {
     // Clear the rect before calling getLocalVisibleRect
     localVisibleRect.setEmpty();
     boolean visibleRect = view.getLocalVisibleRect(localVisibleRect);
-    this.verticalScrolling = vertical;
-    if (vertical) {
-      sizeInScrollingDirection = view.getMeasuredHeight();
-      sizeNotInScrollingDirection = view.getMeasuredWidth();
-      viewportSize = parent.getMeasuredHeight();
-      visibleSize = detachEvent || !visibleRect ? 0 : localVisibleRect.height();
-    } else {
-      sizeNotInScrollingDirection = view.getMeasuredHeight();
-      sizeInScrollingDirection = view.getMeasuredWidth();
-      viewportSize = parent.getMeasuredWidth();
-      visibleSize = detachEvent || !visibleRect ? 0 : localVisibleRect.width();
-    }
-    percentVisibleSize = detachEvent ? 0 : 100.f / sizeInScrollingDirection * visibleSize;
-    if (visibleSize != sizeInScrollingDirection) {
+    height = view.getMeasuredHeight();
+    width = view.getMeasuredWidth();
+    viewportHeight = parent.getMeasuredHeight();
+    viewportWidth = parent.getMeasuredWidth();
+    visibleHeight = detachEvent || !visibleRect ? 0 : localVisibleRect.height();
```

Comment:
since `detachEvent || !visibleRect` is checked many times, maybe it can be standardized in an local var, like `notDrawn`